### PR TITLE
fix scipy, fix pip/apt error

### PR DESCRIPTION
**Pull request by StefanCaldararu**
title, #100

### CODE REVIEW

File: atk.env [hunk 1]
```diff
@@ -1,4 +1,4 @@
 ROS_DISTRO=humble
 
 DEFAULT_APT_DEPENDENCIES="bash vim git git-lfs python3-pip"
-DEFAULT_PIP_REQUIREMENTS=""
+DEFAULT_PIP_REQUIREMENTS="scipy>=1.8"
```
[the remark this answers is not in the document]
Comment (StefanCaldararu) [reply atk.env]: think I did what you meant

File: workspace/src/localization/localization_py/package.xml [hunk 2]
```diff
@@ -14,7 +14,7 @@
 
   <exec_depend>python3-matplotlib</exec_depend>
   <exec_depend>python3-numpy</exec_depend>
-  <exec_depend>python3-scipy</exec_depend>
+  <exec_depend>python3-scipy="1.8"</exec_depend>
```
Comment (AaronYoung5) [inline workspace/src/localization/localization_py/package.xml:17]: nice! Is that allowed? Does that work?

Comment (StefanCaldararu) [reply workspace/src/localization/localization_py/package.xml:17]: @AaronYoung5 yeah seems to work for me. https://stackoverflow.com/questions/75693898/how-to-set-a-specific-version-of-exec-depend-in-ros-package-xml

Comment (AaronYoung5) [reply workspace/src/localization/localization_py/package.xml:17]: I think it should be `<exec_depend version_gte="1.8">python3-scipy</exec_depend>` then, right?

Comment (StefanCaldararu) [reply workspace/src/localization/localization_py/package.xml]: 🤷 worked either way for me... implemented it the way you said

**Review by AaronYoung5 — APPROVED** [review]

**Pull request merged**